src/config-hardware/printers.md

```diff
@@ -0,0 +1,124 @@
+# Setting up Printers
+
+Most printers are plug-and-play in Pop!\_OS. Add a network or USB connected printers in `Settings` ➞ `Printers`. Clicking `Add a Printer` will automatically search for any connected printers. Printers can also be added manually by clicking `Additional Printer Settings`.
+
+![Printer Settings Screen](/images/printers/printer-settings.png)
+
+---
+
+## CUPS and Printer Compatibility
+
+Pop!\_OS uses the Common Unix Printing System (CUPS) print server to manage print jobs, print queues, and recognizes connected printers. While CUPS recognizes a wide range of printer models, printer functionality may vary. Check if your printer is compatible using [OpenPrinter.com](https://www.openprinting.org/printers).
```

Comment:
> Pop!\_OS uses the Common Unix Printing System (CUPS) print server to manage print jobs, print queues, and recognizes connected printers.

There are multiple grammatical issues with this list. `Pop!_OS uses the CUPS print server to recognizes connected printers` doesn't make sense; was that supposed to just be `recognize connected printers`?

If each list item contains its own verb, then `print queues` is missing a verb. You can either repeat the word `manage` in the middle list item (recommended):

> Pop!\_OS uses the Common Unix Printing System (CUPS) print server to manage print jobs, manage print queues, and recognize connected printers.

Or you can split the last item out of the list:

> Pop!\_OS uses the Common Unix Printing System (CUPS) print server to manage print jobs and print queues, and to recognize connected printers.

Reply:
> Pop!_OS uses the CUPS print server to recognizes connected printers doesn't make sense; was that supposed to just be recognize connected printers?

Yea "recognizes" was a typo. I do this when I type too fast. Thanks for catching this.